_tools/install-lookout-latest.sh

```diff
@@ -12,7 +12,7 @@ curl -s --connect-timeout 5 \
     --retry-delay 0 \
     --retry-max-time 40\
     "https://api.github.com/repos/src-d/lookout/releases/latest" \
-  | tee -a ../lookout-install.log \
+  |& tee -a ../lookout-install.log \
```

Comment:
why `&`?

Reply:
allthough it's _bashism_, this is a shortcut that [re-directs stdin and stderr together](https://unix.stackexchange.com/questions/70963/difference-between-2-2-dev-null-dev-null-and-dev-null-21)